XenAdmin/Wizards/RollingUpgradeWizard/PlanActions/RpuUploadAndApplySuppPackPlanAction.cs

```diff
@@ -132,6 +134,10 @@ private void UploadSuppPack(Host master, IXenConnection connection, Session sess
 
                 uploadedSuppPacks.Add(master, poolupdate);
             }
+            else
+            {
+                AddProgressStep(string.Format(Messages.UPDATES_WIZARD_SKIPPING_UPLOAD, suppPack, connection.Name));

```

Comment:
This has the side-effect of adding this progress step even for slaves, where we the upload phase is correctly skipped, but the user does not need to see this entry.